test/cmd/storage.sh

```diff
@@ -27,19 +27,19 @@ run_persistent_volumes_tests() {
 
   ### Create and delete persistent volume examples
   # Pre-condition: no persistent volumes currently exist
-  kube::test::get_object_assert pv "{{range.items}}{{$id_field}}:{{end}}" ''
+  kube::test::get_object_assert pv "{{range.items}}{{${id_field:?}}}:{{end}}" ''
   # Command
-  kubectl create -f test/fixtures/doc-yaml/user-guide/persistent-volumes/volumes/local-01.yaml "${kube_flags[@]}"
-  kube::test::get_object_assert pv "{{range.items}}{{$id_field}}:{{end}}" 'pv0001:'
-  kubectl delete pv pv0001 "${kube_flags[@]}"
-  kubectl create -f test/fixtures/doc-yaml/user-guide/persistent-volumes/volumes/local-02.yaml "${kube_flags[@]}"
-  kube::test::get_object_assert pv "{{range.items}}{{$id_field}}:{{end}}" 'pv0002:'
-  kubectl delete pv pv0002 "${kube_flags[@]}"
-  kubectl create -f test/fixtures/doc-yaml/user-guide/persistent-volumes/volumes/gce.yaml "${kube_flags[@]}"
-  kube::test::get_object_assert pv "{{range.items}}{{$id_field}}:{{end}}" 'pv0003:'
-  kubectl delete pv pv0003 "${kube_flags[@]}"
+  kubectl create -f test/fixtures/doc-yaml/user-guide/persistent-volumes/volumes/local-01.yaml "${kube_flags[@]:?}"
+  kube::test::get_object_assert pv "{{range.items}}{{${id_field:?}}}:{{end}}" 'pv0001:'
+  kubectl delete pv pv0001 "${kube_flags[@]:?}"
```

Comment:
Is the idea that you error out if `kube_flags` is not set or is an empty array? Is that the expected behavior?

Reply:
Error when not set is what we want here.
So it will pass the shellchecks.

Reply:
empty is a good point, we should confirm that we actually don't expect this to ever be an empty array.

Reply:
this _looks_ to be true from walking the invoking scripts.

Reply:
@BenTheElder thanks for your review, do we need to check empty array here, from my tests it seems to be okay.